### PR TITLE
#1 Add scroll arrows

### PR DESCRIPTION
**Pull request by samaSiriwardhanage**
Resolves #X

This is used to indicate to visitors that it is possible to select other spotlight videos from the same screen.

### Acceptance Criteria
- [x] Add scroll arrows

### Relevant design files
* None

### Testing instructions
1. None

### DoD
* None

### CODE REVIEW

File: app/static/style.css [hunk 1]
```diff
@@ -348,3 +353,33 @@ p:last-child {
   height: 50px;
   background: center/contain url('close.svg');
 }
+
```
Comment (ghost) [inline app/static/style.css:356]: Remove the empty lines underneath

File: app/static/style.css [hunk 2]
```diff
@@ -348,3 +353,33 @@ p:last-child {
...
+.arrow_right {
+  position: fixed;
+  bottom: 200px;
+  right: 0px;
+  width: 50px;
+  height: 50px;
+  background: center/contain no-repeat url('right-black.svg');
+  background-color: white;
+  z-index: 10;
+}
```
Comment (ghost) [inline app/static/style.css]: Add a newline here.

I hit this often, it's a little quirk of git - but good practice to add.
https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

**Review by ghost — CHANGES_REQUESTED** [review]
Few small syntax changes.

File: app/static/style.css (hunk 1 above)
Comment (samaSiriwardhanage) [reply app/static/style.css:356]: Removed empty lines.

File: app/static/style.css (hunk 2 above)
Comment (samaSiriwardhanage) [reply app/static/style.css]: Added a newline.

**Review by ghost — APPROVED** [review]
Nice work, sorry for the delay on this one!

**Pull request merged**